resources/shaders/deferred_composition.hlsl

```diff
@@ -54,7 +54,7 @@ void main_cs(int3 dispatch_thread_id : SV_DispatchThreadID)
 		// GBuffer contents
 		float3 albedo = gbuffer_albedo_roughness[screen_coord].xyz;
 		const float roughness = gbuffer_albedo_roughness[screen_coord].w;
-		const float3 normal = gbuffer_normal_metallic[screen_coord].xyz;
+		float3 normal = -gbuffer_normal_metallic[screen_coord].xyz;
```

Comment:
tested on RTX machine and the minus on the normal needs to go as it's not needed anymore.